ProcessLib/Process.h

```diff
@@ -185,12 +185,12 @@ class Process
 
     unsigned const _integration_order;
 
-private:
-    GlobalSparsityPattern _sparsity_pattern;
-
     /// Variables used by this process.
     std::vector<std::reference_wrapper<ProcessVariable>> _process_variables;
```

Comment:
you can access process variables via the public function `getProcessVariables()`.